src/triage/component/postmodeling/contrast/utils/file_helpers.py

```diff
@@ -0,0 +1,50 @@
+'''
+File helpers to download matrices from s3:
+
+To run this functions properly is necessary to define the AWS S3 
+credentials. s3fs library will first look for environment variables 
+or a ~/.aws/credentials file. Also, since s3fs is based in boto3, is 
+possible to define S3 credentials in ~/.boto. 
+
+'''
+
+import s3fs
+import pandas as pd
+import pickle
+
+
+def download_s3(path, read_csv=True, model_file=False):
```

Comment:
Two functions in one, split and use better names. Also, the csv part is kind of lame, you are just wrapping around `pandas.read_csv` (and you use this function in a place which you are already using `read_csv`! (see above)